cmd/users.go

```diff
@@ -22,9 +22,12 @@ func (d DeisCmd) UsersList(results int) error {
 		return err
 	}
 
-	d.Printf("=== Users%s", limitCount(len(users), count))
+	d.Printf("=== Users (* admin)%s", limitCount(len(users), count))
```

Comment:
Yeah, that's why I added `=== Users (* admin)`

I can also add it to the docstring as well though.